tremor-script/rebar.config

```
@@ -2,32 +2,35 @@
 
 {plugins, [rebar3_auto]}.
 {cover_enabled, true}.
-{deps, [ lager, jsxd, jsx ]}.
+{deps, [lager, jsxd, jsx]}.
 
-{pre_hooks,
-  [{"(linux|darwin|solaris)", compile, "make -C c_src"},
-   {"(freebsd)", compile, "gmake -C c_src"}]}.
-{post_hooks,
-  [{"(linux|darwin|solaris)", clean, "make -C c_src clean"},
-   {"(freebsd)", clean, "gmake -C c_src clean"}]}.
+{pre_hooks, [
+  {"(linux|darwin|solaris|freebsd)", compile,
+        "sh -c \"cd .. && cargo build -p tremor-script-nif && cp target/debug/libtremor.so tremor-script/priv\""}
```

the `target/debug/libtremor.so` path is wrong. in the build.rs you build it in `target/<TARGET_TRIPLE>/<PROFILE>/libtremor.{so,dll,dylib}`
You want to use the `libpath` file here like so:

```suggestion
        "sh -c \"cd .. && cargo build -p tremor-script-nif && cp $(cat libpath) tremor-script/priv\""}
```

---

the path is correct, `libpath` isn't a thing here

---

according to the `build.rs` above this file is created during compilation.

---

yeah, that one. :D Sorry, i have not been very specific

---

hah no worries I just didn't make the mental connection :) the content of the file was also wrong, had to fix it sneaky files...